app/scripts/controllers/landing.js

```diff
@@ -44,6 +45,11 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        analytics.digitalData.searchFilter = '';
```

Comment:
@OzzieOrca as for testing ERT I was able to begin testing. So far I don't see the page view direct call firing on the home page unless I'm logged in. Also, SSO Guid is not exposed in the data layer when I'm logged in. The logged In data element is coded to populate logged in or logged out based on SSO Guid being exposed. It wasn't working because when copied from MPDX the direct call had a lowercase "l" in it but it's fixed now. It will always say not logged in until we expose SSO. Or I could change the data element rule to look for logged in status rather than reply on SSO being populated. Maybe I'll change that now and test it. I'm free this afternoon to chat more if needed (6/21).